alembic/versions/8384de6b7c50_add_support_for_differing_inverter_and_.py

```diff
@@ -0,0 +1,58 @@
+"""Add support for differing inverter and module capacities
+
+Revision ID: 8384de6b7c50
+Revises: aeea08bcfc42
+Create Date: 2023-04-08 12:26:23.042938
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.sql import table, column
+
+# revision identifiers, used by Alembic.
+revision = '8384de6b7c50'
+down_revision = 'aeea08bcfc42'
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    op.add_column('sites', sa.Column('module_capacity_kw', sa.Float(), nullable=True, comment='The PV module nameplate capacity of the site'))
+    op.add_column('sites', sa.Column('inverter_capacity_kw', sa.Float(), nullable=True, comment='The PV module nameplate capacity of the site'))
+    
+    op.execute("UPDATE sites SET inverter_capacity_kw=capacity_kw")
+
+    # Create a trigger function to set inverter_capacity_kw to capacity_kw
+    op.execute("""
+        CREATE OR REPLACE FUNCTION set_default_inverter_capacity_kw()
```

Comment:
can you add this trigger with sqlalchamy? instead of raw SQL?

Reply:
From what I can gather, it doesn't seem like alembic has support for adding triggers—everything that I can find recommends adding triggers using raw SQL. I might have missed something though—let me know if there's anything that I can do!